cluster/gce/gci/health-monitor.sh

```diff
@@ -108,5 +108,5 @@ if [[ "${component}" == "container-runtime" ]]; then
 elif [[ "${component}" == "kubelet" ]]; then
   kubelet_monitoring
 else
-  echo "Health monitoring for component "${component}" is not supported!"
+  echo "Health monitoring for component \"${component}\" is not supported!"
```

Comment:
I'm pretty sure this is what we previously intended. 

The old code results in `${component}` not being expanded within quotes AND not having any quotes printed in the output.

Having quotes next to it suggests we wanted one of those things, if we just wanted it to be expanded within quotes then the quotes in the middle are totally unnecessary, so I guessed we wanted it to print like:
`Health monitoring for component "foo" is not supported!`

which this now does.